### PR TITLE
Update CHANGELOG for release v0.0.44

### PR DESCRIPTION
**Pull request by bfjelds**
**What this PR does / why we need it**:
Prepare for release v0.0.44

**If applicable**:
- [ ] this PR contains documentation
- [ ] this PR contains unit tests
- [ ] added code adheres to standard Rust formatting (`cargo fmt`)
- [ ] code builds properly (`cargo build`)
- [ ] code is free of common mistakes (`cargo clippy`)
- [ ] all Akri tests succeed (`cargo test`)
- [ ] inline documentation builds (`cargo doc`)
- [ ] version has been updated appropriately (`./version.sh`)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,35 @@
...
+
+To find out more about Akri, check out our [README](https://github.com/deislabs/akri/blob/main/README.md) and start [contributing](https://github.com/deislabs/akri/blob/main/docs/contributing.md) today!
+
+## New Features
+The v0.0.44 release introduces a number of significant improvements!
+
+* Enable Akri for armv7
+* Create separate Helm charts for releases (akri) and merges (akri-dev)
+* Parameterize Helm for udev beyond simple video scenario
+* Expand udev discovery to include subsystems
```
Comment (kate-goldenring) [inline CHANGELOG.md:14]: This is beyond subsystems filtering support. Maybe change to "Expand udev discovery by supporting filtering by udev rules that look up the device hierarchy such as SUBSYSTEMS, ATTRIBUTES, DRIVERS, KERNELS, and TAGS"

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,35 @@
...
+* Parameterize Helm for udev to allow security context
+* Remove requirement for agent to execute in privileged container
+
+View the [full change log](https://github.com/deislabs/akri/compare/v0.0.35...v0.0.44)
+
+## Breaking Changes
+N/A
+
+## Known Issues
+N/A
```
Comment (kate-goldenring) [inline CHANGELOG.md:24]: Should we add supported K3s, K8s, MicroK8s versions here?

File: CHANGELOG.md (hunk 1 above)
Comment (bfjelds) [reply CHANGELOG.md:14]: 👍

File: CHANGELOG.md (hunk 2 above)
Comment (bfjelds) [reply CHANGELOG.md:24]: Something like?

## Validated With
* Kubernetes: v1.19.4
* k3s: v1.18.9+k3s1
* MicroK8s: 1.18/stable

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,35 @@
+# v0.0.44
+
+## Announcing Akri v0.0.44!
+Akri v0.0.44 is a pre-release of Akri.
+
+To find out more about Akri, check out our [README](https://github.com/deislabs/akri/blob/main/README.md) and start [contributing](https://github.com/deislabs/akri/blob/main/docs/contributing.md) today!
+
+## New Features
```
Comment (jiria) [inline CHANGELOG.md:8]: Was the basic RBAC support part of 0.0.35?

**Review by jiria — APPROVED** [review]

File: CHANGELOG.md (hunk 3 above)
Comment (bfjelds) [reply CHANGELOG.md:8]: yeah, just barely.  you can view 0.0.35 release commits here: https://github.com/deislabs/akri/commits/v0.0.35

Comment (jiria) [reply CHANGELOG.md:8]: Nice, thanks.

**Review by kate-goldenring — APPROVED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (bfjelds) [reply CHANGELOG.md:24]: maybe we need to include the known k3s version issue here as well?

Comment (kate-goldenring) [reply CHANGELOG.md:24]: That would be good. It could link to an issue too.

**Pull request merged**